ras_backstage/resources/survey/get_survey_by_short_name.py

```diff
@@ -24,8 +24,21 @@ def get(short_name):
         survey['shortName'] = format_short_name(survey['shortName'])
 
         ce_list = collection_exercise_controller.get_collection_exercises_by_survey(survey['id'])
-
+        GetSurveyByShortName.add_events(ce_list)
```

Comment:
`self.add_events` is a more standard way to write this

Reply:
Why is self. a more standard way to use static methods?

Reply:
Because it's being called from within a bound method by an instance of the class with that static method. Both implementations do the same thing but `self` tends to read better in this context (as it provides a prompt for the reader that may otherwise go to check where this class was imported from, etc.)

Reply:
This method is also static sp you can't use self in this context.

Reply:
Hah! Missed that entirely :)